docs/standard/microservices-architecture/net-core-net-framework-containers/net-core-container-scenarios.md

```diff
@@ -15,15 +15,17 @@ The following is a more detailed explanation of why to choose .NET Core.
 
 ## Developing and deploying cross platform
 
-Clearly, if your goal is to have an application (web application or service) that can run on multiple platforms supported by Docker (Linux and Windows), the right choice is .NET Core, because .NET Framework only supports Windows.
+Clearly, if your goal is to have an application (web app or service) that can run on multiple platforms supported by Docker (Linux and Windows), the right choice is .NET Core, because .NET Framework only supports Windows.
 
 .NET Core also supports macOS as a development platform. However, when you deploy containers to a Docker host, that host must (currently) be based on Linux or Windows. For example, in a development environment, you could use a Linux VM running on a Mac.
 
-[Visual Studio](https://visualstudio.microsoft.com/) provides an integrated development environment (IDE) for Windows and supports Docker development. 
+[Visual Studio](https://www.visualstudio.com/) provides an integrated development environment (IDE) for Windows and supports Docker development.
```

Comment:
Should this be https://visualstudio.com/vs So that it resolves to the specific product?

/cc @CESARDELATORRE

Reply:
Fixed, you're right, the paragraph is about the product, not the site!